### PR TITLE
Changed support from LuaJIT to all Lua version (5.1 — 5.4)

### PR DESCRIPTION
**Pull request by asumbek**
Hello 👋

I've changed some codes in `rawterm.lua` so it'll support Lua 5.1 — 5.4 instead only LuaJIT. And also changes the revision number and added dependencies in the `.rockspec`.

### CODE REVIEW

File: rawterm-1.1.0-2.rockspec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package = "rawterm"
-version = "1.1.0-1"
+version = "1.1.0-2"
```
Comment (emmachase) [inline rawterm-1.1.0-2.rockspec]: Shouldn't this be 1.2.0-1? It's a new feature, not just a rockspec change.

File: rawterm.lua [hunk 2]
```diff
@@ -24,18 +24,32 @@
...
+unsigned short int ws_row;
+unsigned short int ws_col;
+unsigned short int ws_xpixel;
+unsigned short int ws_ypixel;
+]] or [[
+unsigned int ws_row;
+unsigned int ws_col;
+unsigned int ws_xpixel;
+unsigned int ws_ypixel;
+]]
```
Comment (emmachase) [inline rawterm.lua]: Why is the cdef different between the luajit ffi and cffi-lua?

**Comment by emmachase** [comment]
Hi! Thanks for the PR :) I just had a couple questions.

File: rawterm-1.1.0-2.rockspec (hunk 1 above)
Comment (asumbek) [reply rawterm-1.1.0-2.rockspec]: 👌

File: rawterm.lua (hunk 2 above)
Comment (asumbek) [reply rawterm.lua]: `unsigned short int` is invalid in cffi-lua, but valid in LuaJIT's `ffi` library

File: rawterm-1.1.0-2.rockspec (hunk 1 above)
Comment (asumbek) [reply rawterm-1.1.0-2.rockspec]: I've changed the version to `1.2.0-1`

File: rawterm.lua (hunk 2 above)
Comment (emmachase) [reply rawterm.lua]: Doesn't the definition need to be a short though? Should the definition be altered to:
```
unsigned short ws_row;
unsigned short ws_col;
unsigned short ws_xpixel;
unsigned short ws_ypixel;
```

Comment (asumbek) [reply rawterm.lua]: Changed it 👌

**Comment by emmachase** [comment]
Thank you!

**Pull request merged**